olp-cpp-sdk-dataservice-read/src/StreamLayerClientImpl.h

```diff
@@ -34,10 +39,39 @@ class StreamLayerClientImpl {
 
   virtual ~StreamLayerClientImpl();
 
+  virtual bool CancelPendingRequests();
+
+  virtual client::CancellationToken Subscribe(
+      SubscribeRequest request, SubscribeResponseCallback callback);
+
+  virtual client::CancellableFuture<SubscribeResponse> Subscribe(
+      SubscribeRequest request);
+
  private:
```

Comment:
First the nested structure, then a newline, then everything else without newlines.
Some Doxygen comments on the structure will be appreciated.